docs/docs/explanations/advanced-configuration.md

```diff
@@ -416,6 +411,62 @@ Defining a wildcard certificate decreases the amount of Common Name (CN) names y
 </TabItem>
 </Tabs>
 
+### Shared Storage Configuration
+
+Nebari includes shared file systems for the jupyterhub user storage, jupyterhub shared storage, and conda store shared storage. By default, NFS drives are used.
+
+As of Nebari 2024.9.1, alpha support for [Ceph](https://docs.ceph.com/en/latest/) shared file systems as an alternative to NFS is available. The initial benefit of using Ceph is increased read/write performance compared to NFS, but further benefits are expected in future development. Ceph is a distributed storage system which has the potential to provide increased performance, high availability, data redundancy, storage consolidation, and scalability to Nebari.
+
+:::danger
+Do not switch from one storage type to another on an existing Nebari deployment. Any files in the user home directory and conda environments will be lost if you do so! On GCP, all node groups in the cluster will be destroyed and recreated. Only change the storage type prior to the initial deployment.
+:::
+
+Storage is configured in the nebari config file under the storage section.
+
+```yaml
+storage:
+  type: nfs  # efs (default on AWS), cephfs
+  conda_store: 200Gi
+  shared_filesystem: 200Gi
+```
+
+If using the "cephfs" storage type. The block storage backing all ceph storage will be provisioned using a kubernetes storage class. The default storage class will be used unless specified. Premium storage can be used on some cloud providers to back ceph storage for improved performance. The storage class name can be specified in the yaml file. Be aware that premium storage is not accessible on all node types on some cloud providers.
+
+<Tabs>
+  <TabItem label="AWS" value="AWS" default="true">
+
+Premium storage is not available on AWS.
+</TabItem>
+<TabItem label="Azure" value="Azure">
+
+```yaml
+ceph:
+  storage_class_name: managed-premium
```

Comment:
```suggestion
storage:
  ...
  ceph:
    storage_class_name: managed-premium
```
This was not clear to me in the first glance, might be worth updating the other examples

Reply:
The change you've made doesn't work.  ceph is a top level key in nebari config file.

Reply:
I see that I had a wrong explanation.  I fixed it now.